HZN-632 added setTrapProcessor and HZN-637 Initial check-in of files

pavanrp1 opened the pull request:
Changes made for:

HZN-632: added setTrapProcessor in TrapNotificationHandlerDefaultImpl
HZN-637: Initial check-in of files (blueprint-trapd-handler-default.xml, applicationContext-trapDaemon.xml )

pavanrp1 committed:
HZN-632 added setTrapProcessor and HZN-637 Initial check-in of files

soleger commented on features/events/traps/src/main/resources/META-INF/opennms/applicationContext-trapDaemon.xml:
```diff
@@ -23,6 +23,19 @@
   
+  <bean id="trapNotificationHandler" class="org.opennms.netmgt.trapd.TrapNotificationHandler"/>
+  
+  <bean id="trapReceiverSnmp4jImpl" class="org.opennms.netmgt.trapd.TrapReceiverSnmp4jImpl">
```
There is already a TrapReceiverSnmp4jImpl bean in the file. You need to combine your new changes with it.

pavanrp1 replied:
oh yeah, somehow i missed it. Used existing bean.

soleger commented on features/events/traps/src/main/resources/META-INF/opennms/applicationContext-trapDaemon.xml:
```diff
@@ -23,6 +23,19 @@
     <property name="eventReader" ref="eventReader"/>
   </bean>
   
+  <bean id="trapNotificationHandler" class="org.opennms.netmgt.trapd.TrapNotificationHandler"/>
```
This is an interface. You cannot create a new one here. The handler that you should be using is the TrapNotificationHandlerDefaultImpl bean that is defined below.

pavanrp1 replied:
Updated, using TrapNotificationHandlerDefaultImpl.

pavanrp1 committed:
HZN-637: Added getters and setters for the Trapd class and corresponding application context files

soleger committed:
Merge branch 'develop' into feature-refactor-trapd-HZN-637

soleger committed:
Merge branch 'feature-refactor-trapd-HZN-637' of https://github.com/Delmonte3161/opennms into feature-refactor-trapd-HZN-637

soleger committed:
Merge branch 'feature-refactor-trapd' into feature-refactor-trapd-HZN-637

soleger committed:
Changed the blueprint for core/snmp/api so that it deregisters its
ServiceBasedStrategyResolver when the blueprint shuts down so that
it doesn't stay registered unexpectedly in unit tests.

soleger committed:
Whitespace cleanup.
